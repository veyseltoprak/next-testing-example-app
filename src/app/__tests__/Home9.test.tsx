import { render, waitFor, screen } from '@testing-library/react'
import Home from '../page9'
import * as antd from "antd";

// jest.mock("antd", () => {
//     const originalModule = jest.requireActual("antd");
//     return {
//         ...originalModule,
        
//     };
// });
// jest.mock("antd");
// const mockWarning = antd.message.warning as jest.Mock;

it('should render correctly', async () => {
    const { container } = render(<Home />) // ARRANGE
    // await waitFor(()=>{
    //     expect(mockWarning).toHaveBeenCalled();
    // })
    expect(container).toMatchSnapshot();
})


it('should render correctly2', async () => {
    const { container } = render(<Home />) // ARRANGE
    expect(container).toMatchSnapshot();
})


it('should render correctly3', async () => {
    const { container } = render(<Home />) // ARRANGE
    expect(container).toMatchSnapshot();
})


it('should render correctly4', async () => {
    const { container } = render(<Home />) // ARRANGE
    expect(container).toMatchSnapshot();
})