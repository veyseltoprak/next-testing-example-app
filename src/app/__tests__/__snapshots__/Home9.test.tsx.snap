// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `<div />`;

exports[`should render correctly2 1`] = `<div />`;

exports[`should render correctly3 1`] = `<div />`;

exports[`should render correctly4 1`] = `<div />`;
